kratos/geometries/geometry.h

```diff
@@ -3169,6 +3192,42 @@ class Geometry
 
     PointsArrayType mPoints;
 
+    ///@}
+    ///@name Id Bit-Change Operations
+    ///@{
+
+    /// Checks first bit in mId. 0 -> id; 1 -> name
+    bool IsGeometryIdString()
```

Comment:
You don't need this one. The const version is enough.

Reply:
Alright, I remove it.